operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/AnnotationControllerConfiguration.java

```diff
@@ -35,6 +36,10 @@
   public AnnotationControllerConfiguration(Reconciler<R> reconciler) {
     this.reconciler = reconciler;
     this.annotation = reconciler.getClass().getAnnotation(ControllerConfiguration.class);
+    if (annotation == null) {
+      throw new OperatorException("Missing mandatory " +
+          "@ControllerConfiguration annotation from the reconciler:  " + reconciler);
```

Comment:
Nitpick: use the class name of the annotation instead of hardcoding it?

Reply:
good point, fixed, thx